css/modern.css

```diff
@@ -1755,9 +1765,664 @@ table#collection-view-contents tbody tr td {
     background-color: var(--highlight) !important;
 }
 
+
+/* Quizes */
```

Comment:
Spelling. Also, "modern" UI quizzes or old quizzes (please clarify)?

Reply:
I believe it is old quizes